sys/drv/atkbdc.c

```diff
@@ -61,28 +66,27 @@ static void write_data(resource_t *regs, uint8_t byte) {
   bus_write_1(regs, KBD_DATA_PORT, byte);
 }
 
-static int scancode_read(vnode_t *v, uio_t *uio, int ioflag) {
-  atkbdc_state_t *atkbdc = devfs_node_data(v);
-  int error;
+static void atkbdc_thread(void *arg) {
```

Comment:
I'm afraid that function with that name really needs a comment of explanation

Reply:
Done